.github/workflows/ci.yml

```diff
@@ -0,0 +1,112 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  release:
+    types: [published]
+
+jobs:
+  Build:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        dc:
+          - ldc-latest
+          - dmd-latest
+        arch:
+          - x86_64
+        include:
+          # windows x86
+          - os: windows-latest
+            arch: x86
+            dc: ldc-latest
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Setup D
+        uses: dlang-community/setup-dlang@v1
+        with:
+          compiler: ${{ matrix.dc }}
+
+      # Build
+
+      - name: Build
+        run: |
+          dub build --build=release --config=client --arch=${{ matrix.arch }}
+          dub build --build=release --config=server --arch=${{ matrix.arch }}
+
+      # Tests
+
+      - name: Linux Tests
+        if: contains(matrix.os, 'ubuntu')
+        run: |
+          ./run_tests.sh
+        working-directory: tests
+        shell: bash
+
+      - name: Windows and MacOS Tests
+        if: contains(matrix.os, 'windows') || contains(matrix.os, 'macos')
+        run: |
+          ./run_tests.sh
+        working-directory: tests
+        shell: bash
+        continue-on-error: true
+
+
+      # Package Release
+
+      - name: TarGZ MacOS and Linux
+        if: github.event_name == 'release' && !contains(matrix.os, 'windows') && contains(matrix.dc, 'ldc')
+        uses: thedoctor0/zip-release@0.4.2
+        with:
+          filename: dcd.tar.gz
+          path: "dcd-client dcd-server"
+
+      - name: Zip Windows
+        if: github.event_name == 'release' && contains(matrix.os, 'windows') && contains(matrix.dc, 'ldc')
+        uses: thedoctor0/zip-release@0.4.2
+        with:
+          filename: dcd.zip
+          path: "dcd-client.exe dcd-server.exe"
+
+      # Release
+
+      - name: Release Linux
+        if: github.event_name == 'release' && contains(matrix.os, 'ubuntu') && contains(matrix.dc, 'ldc')
+        uses: WebFreak001/upload-asset@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OS: linux
+        with:
+          file: dcd.tar.gz
+          filename: dcd-${TAG}-${OS}-${{ matrix.arch }}.tar.gz
```

Comment:
TAG does not match previous naming convention, use `${TAG_RAW}` instead

Reply:
Fixed in https://github.com/dlang-community/DCD/pull/637/commits/a44dd86720af88c224b22c58b48cddadf2ca03f2